src/main/java/org/symphonyoss/integration/webhook/salesforce/SalesforceConstants.java

```diff
@@ -70,6 +70,8 @@ private SalesforceConstants(){}
   public static final String HAS_OWNER_AT_SYMPHONY = "hasOwnerAtSymphony";
   public static final String URL_ICON_INTEGRATION = "URLIconIntegration";
   public static final String ICON_CROWN = "IconCrown";
+  public static final String NAME_AND_EMAIL = "NameAndEmail";
```

Comment:
We should follow a pattern for these values, or they are lowerCamelCase or UpperCamelCase.